src/state/dates/selectors.js

```diff
@@ -3,3 +3,13 @@ export const getStartDate = state =>
 
 export const getEndDate = state =>
   state.get("dates").get("endDate");
```

Comment:
I just noticed there are other places you are using nested `.get` calls. Leave this as-is here but I'll file a new issue to fix this.

Reply:
Ok sounds good thanks!!!